sources/PTYSession.m

```diff
@@ -3503,10 +3491,8 @@ - (void)scheduleUpdateIn:(NSTimeInterval)timeout {
 
 - (void)doAntiIdle {
     NSTimeInterval now = [NSDate timeIntervalSinceReferenceDate];
-    if (now >= _lastInput + 60) {
-        [_shell writeTask:[NSData dataWithBytes:&_antiIdleCode length:1]];
-        _lastInput = now;
-    }
+    [_shell writeTask:[NSData dataWithBytes:&_antiIdleCode length:1]];
```

Comment:
The if statement ensured that it actually only activated during idle times. As this is written, it will write when not idle. The original behavior seems better, since  writing the anti-idle code while you're busy typing a command would (perhaps invisibly) corrupt your work.